astropy/modeling/utils.py

```diff
@@ -493,3 +493,190 @@ def combine_labels(left, right):
 
     return left + right
 
+
+def ellipse_extent(a, b, theta):
+    """
+    Calculates the extent of a box encapsulating a rotated 2D ellipse.
+
+    Parameters
+    ----------
+    a : float
+        Major axis
+    b : float
+        Minor axis
+    theta: float
+        Rotation angle in radians.
+
+    Returns
+    -------
+    image : array
+            The evaluated model on the input image or coordinates. If 
```

Comment:
Indentation too large here.
Are you using tabs?
Always use 4-space indentation.